s3store/s3store.go

```diff
@@ -517,6 +501,33 @@ func (store S3Store) ConcatUploads(dest string, partialUploads []string) error {
 	return store.FinishUpload(dest)
 }
 
+func (store S3Store)ListAllParts(id string) (parts []*s3.Part, err error) {
```

Comment:
Is there a way to keep `TestListParts` working (which should be renamed to `TestListAllParts` anyway)? Otherwise I need to test `listAllParts` through some of the other functions..